### PR TITLE
CASM-2423 Add documentation for Grafterm tool

### PR DESCRIPTION
**Pull request by itsvijaybs**
This change is needed for: https://connect.us.cray.com/jira/browse/CASMMON-98 

image

### CODE REVIEW

File: operations/system_management_health/Grafterm.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Grafterm
+
+Visualize metrics dashboards on the terminal, like a simplified and minimalist version of **Grafana** for terminal.
+
+The utility(script) can be found in the /opt/cray/platform-utils directoty in all NCNs.
```
Comment (bklei) [inline operations/system_management_health/Grafterm.md:5]: 'directory'

File: operations/system_management_health/Grafterm.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ncn-m001# ./grafterm.sh --help
+```
+
+### List available terminal dashboards
+
+```bash
+ncn-m001# ./grafterm.sh --list
+```
+
+### simple
```
Comment (bklei) [inline operations/system_management_health/Grafterm.md:23]: Maybe 'Default Usage'? Could also say what the default days and refresh settings are...

File: operations/system_management_health/Grafterm.md (hunk 1 above)
Comment (itsvijaybs) [reply operations/system_management_health/Grafterm.md]: > 'directory'

Addressed.

File: operations/system_management_health/Grafterm.md (hunk 2 above)
Comment (itsvijaybs) [reply operations/system_management_health/Grafterm.md]: > Maybe 'Default Usage'? Could also say what the default days and refresh settings are...

Addressed. Thanks!

**Review by bklei — APPROVED** [review]

**Review by denniswalker — APPROVED** [review]

**Pull request merged**